test/Constraints/generics.swift

```diff
@@ -52,9 +52,9 @@ func generic_metatypes<T : SomeProtocol>(_ x: T)
 }
 
 // Inferring a variable's type from a call to a generic.
-struct Pair<T, U> { } // expected-note 5 {{'T' declared as parameter to type 'Pair'}}
+struct Pair<T, U> { } // expected-note 4 {{'T' declared as parameter to type 'Pair'}} expected-note {{'U' declared as parameter to type 'Pair'}}
```

Comment:
That's right, everything is producing the same number of notes but since one of the FIXMEs in the test if fixed now it produces correct node regarding U instead of T.